### ISSUE TITLE
No (additional) cloud-side encryption?

### ISSUE THREAD

**Issue by mih** [datalad/openneuropet#3]
OpenNeuroPET plans to upload encrypted data only. I understood this as client-side/pre-upload encryption. It remains to be decided how the encryption keys are implemented (git-annex?) and exchanged.

**Comment by adswa** [comment]
@cpernet talked about encryption at REST on the Azure servers, which presumably gets decrypted prior (?) to transport.

**Comment by mslw** [comment]
I also understood it as server side encryption, where access (by login/token) is sufficient to obtain the decrypted content. Encryption in transfer probably refers to TLS.

**Comment by mslw** [comment]
General docs on [Azure Storage encryption for data at rest](https://learn.microsoft.com/en-us/azure/storage/common/storage-service-encryption) found by @adswa

**Comment by CPernet** [comment]
yep, I'll just upload files as it is and let Azure deal with encryption
